cfgov/v1/config.py

```diff
@@ -0,0 +1,12 @@
+ # These messages are manually mirrored on the
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/113c70c56a83dd183eb99bfa088e5324e1aad546